docs/php/how-to-define-workflow-return-values-in-php.md

```diff
@@ -0,0 +1,18 @@
+---
+id: how-to-define-workflow-return-values-in-php
+title: Define Workflow return values in PHP
+sidebar_label: Workflow return values
+---
+
+A Workflow method returns a Generator.
+To properly typecast the Workflow's return value in the client code, use the `#[ReturnType()]` annotation.
+
+```php
+#[WorkflowInterface]
+interface FileProcessingWorkflow {
```

Comment:
I really want to see "YourWorkflowInterface" or something similarly generic instead of FileProcessing.